### ISSUE TITLE
Updating in background?

### ISSUE THREAD

**Issue by zackify** [cristianoalves92/react-native-live-activity#1]
How would you propose updating the activities in the background?

could you run a background task in react native that calls updateActivity periodically?

Or would you have to setup the push notification flow

**Comment by anthlasserre** [comment]
If you want to support background tasks at some point I think you can use that library https://github.com/Rapsssito/react-native-background-actions

**Comment by anthlasserre** [comment]
I can confirm that this is working great.

**Comment by mrados7** [comment]
@anthlasserre I will give this a try, I thought you cannot update anything in background from javascript as it is being "killed" when using React Native. 
One solution that I know will work for a fact is to use Apple APN service to send live notification update.
https://developer.apple.com/documentation/activitykit/updating-live-activities-with-activitykit-push-notifications

**Comment by anthlasserre** [comment]
Yeah indeed, the best solution ever to update Live Activity widgets is to populate the values through push notifications payload. For instance 3rd parties like Braze & OneSignal already support these features.

**Comment by anthlasserre** [comment]
As this issue is outside the scope of React Native, should we close it, @mrados7 ?

**Comment by mrados7** [comment]
@anthlasserre I agree, you can close this one.

**Comment by anthlasserre** [comment]
@zackify ?

**Comment by megabeehothanhminh** [comment]
Hello. I have a issue about image url in live activity. I show image with url in simulator it work. but on device real not work. 
You can help me? Handle issue this?. Thank you!

**Comment by Jamal-Ud-Din-hub** [comment]
@anthlasserre can we update it using push notification?

**Comment by anthlasserre** [comment]
> [@anthlasserre](https://github.com/anthlasserre) can we update it using push notification?

Yes, we can, thanks to [broadcast push notifications](https://developer.apple.com/documentation/usernotifications/setting-up-broadcast-push-notifications) that Apple delivered this year. This is a recommended process if you want to update data from an external service. I think we can close that issue by the way. What do you think @cristianoalves92 @G2Jose ?
